<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Spielfenster</class>
 <widget class="QDialog" name="Spielfenster">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>601</width>
    <height>531</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QPushButton" name="Spiel_beenden">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>121</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Spiel Beenden</string>
   </property>
  </widget>
  <widget class="QLabel" name="SpinningWheel1">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>240</y>
     <width>71</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="SpinningWheel2">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>240</y>
     <width>61</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="SpinningWheel3">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>250</y>
     <width>81</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="Aktion_Line_Edit">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>380</y>
     <width>251</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="Name_Line_Edit">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>460</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
